diff --git a/dist/src/vitest-test-runner.js b/dist/src/vitest-test-runner.js
index 84f1ca026503b8e92b7adc952ec6c6a401943656..d2de85bade53239df779386b516b2a3aad679eae 100644
--- a/dist/src/vitest-test-runner.js
+++ b/dist/src/vitest-test-runner.js
@@ -2,6 +2,7 @@ import { fileURLToPath } from 'url';
 import path from 'path';
 import semver from 'semver';
 import fs from 'fs';
+import { inspect } from 'node:util';
 import { INSTRUMENTER_CONSTANTS, } from '@stryker-mutator/api/core';
 import { commonTokens, tokens, } from '@stryker-mutator/api/plugin';
 import { DryRunStatus, toMutantRunResult, determineHitLimitReached, TestStatus, } from '@stryker-mutator/api/test-runner';
@@ -92,7 +93,7 @@ export class VitestTestRunner {
             project.config.browser.screenshotFailures = false;
         });
         if (this.log.isDebugEnabled()) {
-            this.log.debug(`vitest final config: ${JSON.stringify(this.ctx.config, null, 2)}`);
+            this.log.debug(`vitest final config: ${inspect(this.ctx.config, { depth: 3, maxArrayLength: 20, maxStringLength: 1000, customInspect: false })}`);
         }
     }
     async dryRun(options) {
